src/app/navigation/examples/application-launcher-example.component.html

```diff
@@ -0,0 +1,61 @@
+<ul class="nav navbar-nav">
+</ul>
+<ul class="nav navbar-nav navbar-right navbar-iconic">
+    <li class="dropdown">
+        <pfng-application-launcher
+        [label] ="'Application Launcher test'"
+        [isDisabled] = "false"
+        [isList]="false"
+        [items]="navigationItems"
+        [hiddenIcons] = "false">
+      </pfng-application-launcher>
```

Comment:
![screen shot 2018-01-25 at 5 40 34 pm](https://user-images.githubusercontent.com/17481322/35416421-08a47c76-01f7-11e8-979b-a20c3808c08a.png)
The app launcher doesn't appear aligned with the other dropdowns.